src/mlpack/tests/main_tests/perceptron_test.cpp

```diff
@@ -1,21 +1,18 @@
 /**
- * @file perceptron_test.cpp
- * @author Manish Kumar
- *
- * Test mlpackMain() of perceptron_main.cpp.
- *
- * mlpack is free software; you may redistribute it and/or modify it under the
- * terms of the 3-clause BSD license.  You should have received a copy of the
- * 3-clause BSD license along with mlpack.  If not, see
- * http://www.opensource.org/licenses/BSD-3-Clause for more information.
- */
+  * @file perceptron_test.cpp
+  * @author B Kartheek Reddy
```

Comment:
Why did you remove Manish's name? He wrote many of these tests and we should leave them in.

Reply:
Sorry for the inconvenience. I think I did a mistake in resolving merge conflicts. I misunderstood. I thought I have to push my file and then afterwards you will merge it with Manish's tests. Now I understand. Don't worry I will correct it now by adding my tests to Manish's Perceptron Test and push that file. I will remember this next time. Thanks for Reviewing.

Reply:
No worries, when you get it fixed I'll take a look at the perceptron tests and we can finish the review process. :)